frontend/simple/js/idb-flume-log.js

```diff
@@ -0,0 +1,172 @@
+import localforage from 'localforage'
```

Comment:
Issue #160 means this file shouldn't be called `idb-flume-log.js`... if idb means IndexedDB. On Safari Localforage will use WebSQL, and in the future, per #160, we might use something else entirely (not based on localforage).

Therefore, could you rename this file and create the abstraction support needed for #160 to go forward?